admin/src/components/CMEditView/RightLinksCompo/Summary/PreviewChecks/index.js

```diff
@@ -26,7 +26,10 @@ const SeoChecker = ({ checks }) => {
   return (
     <Box paddingTop={4}>
       <Typography variant="omega" fontWeight="semiBold">
-        SEO Summary
+	  {formatMessage({
+		  id: getTrad('SEOComponent.title.summary'),
```

Comment:
You are not creating translations for `SEOComponent.title.summary` in en and fr but a `SEO Summary` one instead.

Can you please remove the `SEO Summary`, add the `SEOComponent.title.summary` in both fr and en?